### ISSUE TITLE
Provider fails due to missing dependency

### ISSUE THREAD

**Issue by plumdog** [isotoma/ses-smtp-credentials-cdk#2]
With 1.0.2, get an error like:
```
16:21:30 | CREATE_FAILED        | Custom::SesSmtpCredentials | Credentials/Resource/Default
Failed to create resource. Error: Cannot find module 'strftime'
Require stack:
- /var/task/index.js
- /var/runtime/UserFunction.js
- /var/runtime/index.js
```

Inspection of the installed files shows:
```
$ ls node_modules/ses-smtp-credentials-cdk/provider
index.ts  jest.config.js  lib  package.json  package-lock.json  tsconfig.json
```
So something has gone a bit awry in the build and publish steps, because there's no `node_modules/` there, which I would expect to find.

I also tried importing like:
```typescript
import { SesSmtpCredentials } from 'ses-smtp-credentials-cdk/lib';
```
which gets the same error, and:
```
$ ls node_modules/ses-smtp-credentials-cdk/lib/provider
index.js
```

**Comment by plumdog** [comment]
Intended approach: use CDK's `NodejsFunction`.

**Comment by plumdog** [comment]
Fixed by #4
